org.civicrm.bootstrapcivicrm/bootstrapcivicrm.php

```diff
@@ -123,22 +123,26 @@ function bootstrapcivicrm_civicrm_alterSettingsFolders(&$metaDataFolders = NULL)
 }
 
 /**
- * Implementation of hook_civicrm_pageRun
+ * Implementation of hook_civicrm_coreResourceList
  */
-function bootstrapcivicrm_civicrm_pageRun($page) {
-  if (!(isset($_GET['snippet']) && $_GET['snippet'] == 'json')) {
-    CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivicrm', 'css/bootstrap.css');
+function bootstrapcivicrm_civicrm_coreResourceList(&$items, $region) {
+  if ($region == 'html-header') {
+    CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivicrm', 'css/bootstrap.css', -50, 'html-header');
     CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrapcivicrm', 'js/radio-checkbox.js');
     CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrapcivicrm', 'js/add-missing-date-addons.js');
+    CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrap', 'js/transition.js', 1000, 'html-header');
```

Comment:
It's ok to place scripts in other regions via this hook. The reason for the conditional is just to filter out edge-cases where the hook might be called when CiviCRM is *not* loading its main styles and scripts.